tests/run.test.js

```diff
@@ -317,6 +318,18 @@ test('should fail if jankiness score is out of budget', async () => {
     expect(process.exit).toBeCalledWith(1)
 })
 
+test('should include header when using sauce rest API', async () => {
+    await handler({
+        user: 'foo',
+        key: 'bar',
+        site: 'mypage',
+        metric: ['load', 'speedIndex'],
+        traceLogs: true,
+        logDir: '/foo/bar'
+    })
+    expect(lastInstance.args[0].headers).toMatchSnapshot()
```

Comment:
This becomes problematic for people that use a Windows machine, I would suggest:

```suggestion
    expect(lastInstance.args[0].headers['User-Agent']).toContain('Speedo/1.2.4 (nodejs')
```

or mock the platform property of the `process` object.

Reply:
I don't think this will be problem on any OS platform, as we mocked constants in the beginning of the file.
```
jest.mock('../src/constants', () => ({
    TUNNEL_SHUTDOWN_TIMEOUT: 100,
    REQUIRED_TESTS_FOR_BASELINE_COUNT: 10,
    JANKINESS_METRIC: 'jankiness',
    SPEEDO_API_UA: 'Speedo/1.2.4 (nodejs darwin)'
}))
```

Reply:
that makes sense